static/sass/_vars.scss

```diff
@@ -21,6 +21,7 @@ $content-padding: 16px;
 $default-border-radius: 3px;
 
 $app-drawer-width: 200px;
+$header-height: 202px;
```

Comment:
Yea, I think `$header-height: 135px`; is less confusing. Then add another var for the banner? 

Also what happens when we don't have a banner? This will need to account for that.